src/App.jsx

```diff
@@ -11,6 +12,7 @@ function App() {
       <Header/>
       <Routes>
         <Route path="/" element={<Home />} />
```

Comment:
I'm wondering whether we could maybe have another route in here for /stories which also takes us to the Home component? I'm just thinking from a user point of view, once they're at stories/1 for example, they may want to delete the 1 just to get back to all stories, but at the moment that would just be a blank page